src/main/java/app/studnicki/ox/Config.java

```diff
@@ -6,12 +6,15 @@
 /**
  * Configuration class for the whole application. It is implemented as a singleton (enum property).
  * Config can retrieve values for specified keys from properties files and change language of them.
+ * @author Aleksander Studnicki
  */
 enum Config {
   INSTANCE;
 
   private Locale locale = Locale.forLanguageTag("en");
 
+  final int MINIMUM_DIMENSION = 3;
```

Comment:
It's up to you but conventions say that only constants (final static) should be camel cased.

Reply:
Yeah, I just forgot static keyword.